src/test/java/org/assertj/core/api/Assertions_assertThatObject_Test.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2018 the original author or authors.
```

Comment:
if we really want to have a Copyright in the doc, then it's 2012-2019.

Reply:
we will get there with the latest parent pom: https://github.com/joel-costigliola/assertj-maven-parent-pom/blob/master/pom.xml#L387